polytope/polytope.py

```diff
@@ -570,6 +564,18 @@ def _rotate(polyreg, i=None, j=None, u=None, v=None, theta=None, R=None):
     return R
 
 
+def _Givens_rotation_matrix(i, j, theta, N):
```

Comment:
Why not make this part of the public API (and delete the leading `_`)?

Whether or not it is part of the public API, there should be unit tests created for it.